tests/ValidationJSON.test.ts

```diff
@@ -0,0 +1,102 @@
+/*******************************************************************************
+
+    Test that validation with JSON schema
+
+    Copyright:
+        Copyright (c) 2020 BOS Platform Foundation Korea
+        All rights reserved.
+
+    License:
+        MIT License. See LICENSE for details.
+
+*******************************************************************************/
+
+import * as assert from 'assert';
+import { Validator, ITransaction, IEnrollment } from '../src/modules/data/validator';
+import { Enrollment } from "../src/modules/data";
+
+describe ('Test that validation with JSON schema', () =>
+{
+    it ('Test that validation of Transaction', () =>
+    {
+        assert.throws(() =>
+        {
+            // When attribute `type` is not present
+            Validator.isValidOtherwiseThrow<ITransaction>("Transaction", {
+                "inputs": [],
+                "outputs": []
+            });
+        }, new Error("Validation failed: Transaction - should have required property 'type'"));
+
+        assert.throws(() =>
+        {
+            // When attribute `inputs` is not an array
+            Validator.isValidOtherwiseThrow<ITransaction>("Transaction", {
+                "type": 1,
+                "inputs": {},
+                "outputs": []
+            });
+        }, new Error("Validation failed: Transaction - should be array"));
+
+        // When attribute `type` is not present
+        assert.ok(!Validator.isValidOtherwiseNoThrow<ITransaction>("Transaction", {
+            "inputs": [],
+            "outputs": []
+        }));
+
+        // When attribute `inputs` is not an array
+        assert.ok(!Validator.isValidOtherwiseNoThrow<ITransaction>("Transaction", {
+            "type": 1,
+            "inputs": {},
+            "outputs": []
+        }));
+
+        // When everything is normal
+        assert.ok(Validator.isValidOtherwiseThrow<ITransaction>("Transaction", {
+            "type": 1,
+            "inputs": [],
+            "outputs": [
+                {
+                    "value": "400000000000",
+                    "address": "GA3DMXTREDC4AIUTHRFIXCKWKF7BDIXRWM2KLV74OPK2OKDM2VJ235GN"
+                }
+            ]
+        }));
```

Comment:
Input value is required on normal case .

Reply:
``` typescript
    {
            "type": 0,
            "inputs": [
                {
                    "previous": "0x5d7f6a7a30f7ff591c8649f61eb8a35d034824ed5" +
                        "cd252c2c6f10cdbd2236713dc369ef2a44b62ba113814a9d819" +
                        "a276ff61582874c9aee9c98efa2aa1f10d73",
                    "index": 1,
                    "signature": "0x07557ce0845a7ccbba61643b95e310bd3ae06c41" +
                        "fab9e8761ff3b0e5d28a5d625a3b951223c618910b239e7b779" +
                        "c6c671252a78edff4d0f37bdb25982e4f4228"
                }
            ],
            "outputs": [
                {
                    "value": "400000000000",
                    "address": "GA3DMXTREDC4AIUTHRFIXCKWKF7BDIXRWM2KLV74OPK2" +
                        "OKDM2VJ235GN"
                }
            ]
        }
```